infra/main.bicep

```diff
@@ -0,0 +1,483 @@
+  }
+  kind: 'FormRecognizer'
+  identity: {
+    type: 'None'
+  }
+  properties: {
+    networkAcls: {
+      defaultAction: 'Allow'
+      virtualNetworkRules: []
+      ipRules: []
+    }
+    publicNetworkAccess: 'Enabled'
+  }
+}
+
+resource contentSafety 'Microsoft.CognitiveServices/accounts@2022-03-01' = {
+  name: contentSafetyName
+  location: location
+  sku: {
+    name: 'S0'
+  }
+  kind: 'ContentSafety'
+  identity: {
+    type: 'None'
+  }
+  properties: {
+    networkAcls: {
+      defaultAction: 'Allow'
+      virtualNetworkRules: []
+      ipRules: [] 
+    }
+  }
+}
+
+resource eventGridSystemTopic 'Microsoft.EventGrid/systemTopics@2021-12-01' = {
+  name: eventGridSystemTopicName
+  location: location
+  properties: {
+    source: storageAccount.id
+    topicType: 'Microsoft.Storage.StorageAccounts'
+  }
+}
+
+resource eventGridSystemTopicNameBlobEvents 'Microsoft.EventGrid/systemTopics/eventSubscriptions@2021-12-01' = {
+  parent: eventGridSystemTopic
+  name: 'BlobEvents'
+  properties: {
+    destination: {
+      endpointType: 'StorageQueue'
+      properties: {
+        queueMessageTimeToLiveInSeconds: -1
+        queueName: queueName
+        resourceId: storageAccount.id
+      }
+    }
+    filter: {
+      includedEventTypes: [
+        'Microsoft.Storage.BlobCreated'
+        'Microsoft.Storage.BlobDeleted'
+      ]
+      enableAdvancedFilteringOnArrays: true
+      subjectBeginsWith: '/blobServices/default/containers/${blobContainerName}/blobs/'
+    }
+    labels: []
+    eventDeliverySchema: 'EventGridSchema'
+    retryPolicy: {
+      maxDeliveryAttempts: 30
+      eventTimeToLiveInMinutes: 1440
+    }
+  }
+}
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-09-01' = {
+  name: storageAccountName
+  location: location
+  kind: 'StorageV2'
+  sku: {
+    name: 'Standard_GRS'
+  }
+  resource storageAccountNameDefaultBlob 'blobServices' = {
+    name: 'default'
+    resource storageAccountNameDefaultBlobContainer 'containers' = {
+      name: blobContainerName
+      properties: {
+        publicAccess: 'None'
+      }
+    }
+    resource storageAccountNameDefaultConfig 'containers' = {
+      name: 'config'
+      properties: {
+        publicAccess: 'None'
+      }
+    }
+  }
+}
+
+resource storageAccountNameDefault 'Microsoft.Storage/storageAccounts/queueServices@2022-09-01' = {
+  parent: storageAccount
+  name: 'default'
+  properties: {
+    cors: {
+      corsRules: []
+    }
+  }
+}
+
+resource storageAccountNameDefaultDocProcessing 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: storageAccountNameDefault
+  name: 'doc-processing'
+  properties: {
+    metadata: {}
+  }
+  dependsOn: []
+}
+
+resource storageAccountNameDefaultDocProcessingPoison 'Microsoft.Storage/storageAccounts/queueServices/queues@2022-09-01' = {
+  parent: storageAccountNameDefault
+  name: 'doc-processing-poison'
+  properties: {
+    metadata: {}
+  }
+  dependsOn: []
+}
```

Comment:
main.bicep should have all the outputs needed to run the application.  See other azd templates for examples